txdbus/authentication.py

```diff
@@ -141,7 +146,7 @@ def _auth_DATA(self, line):
                 self.sendAuthMessage(b'ERROR ' + str(e).encode())
 
     def _auth_ERROR(self, line):
-        log.msg('Authentication mechanism failed: ' + line)
+        log.msg('Authentication mechanism failed: ' + line.decode())
```

Comment:
Use `.decode("ascii", "replace")` to prevent decoding errors from the auth message causing havoc